servers/partner-ci/9.0/contrail.yaml

```diff
@@ -448,20 +441,23 @@
       - test-property
     builders:
       - get-refs
+      - shell: |
+          if [ ! -z "${{COMMON_PROPS}}" ]; then
+            echo > common.properties
+            for param in ${{COMMON_PROPS}}; do
+              echo $param >> common.properties
+            done
+          fi
+          if [ ! -z "${{SPECIFIC_PROPS}}" ]; then
```

Comment:
Seems like logic mistake:
If COMMON_PROPS doesn't exist, then common.properties won't be erased, and then will be injected with invalid properties. Please clean *.properties in for each case